### ISSUE TITLE
Suggestion: Migrate to newer technologies

### ISSUE THREAD

**Issue by VibhorCodecianGupta** [scorelab/Bassa#614]
### Expected Behavior
Smooth installation process without any dependency hassle. Setting up the project should require minimal steps.

### Current Behavior
While setting up Bassa recently on my machine, I came across a couple of serious issues. 

 - [] First and foremost, the `README.md` is missing the command `pip3 install -r requirements.txt`. For someone new

### Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment
* Environment name and version (e.g. Chrome 39, node.js 5.4): 
* Operating System and version (desktop or mobile): MacBook Air, OSX 10.13.6

**Comment by VibhorCodecianGupta** [comment]
Unable to login as well. On the `develop` branch of this repository.

**Comment by vivonk** [comment]
@VibhorCodecianGupta - @agentmilindu was thinking to shift from bower and gulp to webpack so that will reduce this error. 
I have also faced a lot of errors with `node sass` so workaround can replace it will something else.
I haven't faced any problem with MySQLdb and libmysqlclient till the date but I guess it' making problem on Macbooks.
We can have a good discussion about upgrading our techs.

**Comment by VibhorCodecianGupta** [comment]
@vivonk absolutely. React with `webpack` and `babel` plugins is the brightest future for Bassa in my opinion.

**Comment by vivonk** [comment]
@VibhorCodecianGupta I guess shifting from angularjs to React might not be a right decision at this stage. Alternatively, we can have a branch where we can work on this.
@agentmilindu @rehrumesh can provide better insights into this :smile:

**Comment by VibhorCodecianGupta** [comment]
Yes, sure. It can be initiated as a different release altogether, in a separate branch. If it's convincing enough to replace the current architecture, only then it could be considered to go ahead. On the other hand, we could also discuss migrating to Angular2/Angular4 instead of React.

**Comment by lakinduakash** [comment]
s

**Comment by lakinduakash** [comment]
@vivonk Do we have a new branch for migrating?
